### ISSUE TITLE
Two small questions about the code 

### ISSUE THREAD

**Issue by albert-ba** [NoamRosenberg/AutoML#41]
Hi, first I wanted to say that you are doing an amazing job. 
I have two small questions, if you don't mind:
1.what is going on in this line:
    `out = torch.cat([self.conv_1(x), self.conv_2(x[:,:,1:,1:])], dim=1)`
link: https://github.com/NoamRosenberg/AutoML/blob/master/operations.py#L95
couldn't find anything like that in the paper.

2. In  https://github.com/NoamRosenberg/AutoML/blob/master/cell_level_search.py#L142
why you are doing this action: `states[-self.block_multiplier:]`

Thanks a lot!

**Comment by HankKung** [comment]
Hello, thanks for your feedback!

1. FactorizedReduce is implemented as two stride 2 Convs concatenation for a larger field of view. These two Convs' kernels always see a different area of an image from each other by stride 2 and x[:,:,1:,1:] cropping. Sketch out the operation and you will see!
In additionally, the paper actually mentioned that they reused the cell operations set from [DARTS](https://github.com/quark0/darts/blob/f276dd346a09ae3160f8e3aca5c7b193fda1da37/cnn/operations.py#L90). The description is on the top of Section 3.

2. The output of a cell is a concatenation of each block's output. Each block's output is gathered at [here](https://github.com/NoamRosenberg/AutoML/blob/a06bc1a286dfd1149784e9ccea461cdb58cbf838/cell_level_search.py#L140). Hence, the last #self.block_multiplier elements are what we need to concatenate.
